lib/hal/impression/rpihal.ex

```diff
@@ -216,6 +258,10 @@ defmodule Inky.Impression.RpiHAL do
   # 0b00000010 = DC-DC converter, 0 = off, 1 = on
   # 0b00000001 = Soft reset, 0 = Reset, 1 = Normal (Default)
   defp set_panel(state), do: write_command(state, @psr, [0b11101111, 0x08])
+  # NOTE: Tried several alternatives below
+  # defp set_panel(state), do: write_command(state, @psr, [Bitwise.<<<(0b11, 6), 0x08])
+  # defp set_panel(state), do: write_command(state, @psr, [0b11000000, 0x08])
+  # TODO: Figure out how panel workks and whether it needs adjustment.
```

Comment:
I experimented with changing the commands written to set the panel. However, I'm not sure what the panel is or does. I got varied results with my experiments and none seemed better than the existing code: `write_command(state, @psr, [0b11101111, 0x08])`.